code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/file/property/PropertyFile.xtend

```diff
@@ -0,0 +1,118 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.file.property
+
+import java.io.BufferedWriter
+import java.io.FileWriter
+import java.io.StringWriter
+import java.util.function.BiConsumer
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.io.File
+import de.fhdo.lemma.model_processing.code_generation.container_base.util.Util
+
+/**
+ * Represents a property file in Java's
+ * [properties format](https://docs.oracle.com/javase/10/docs/api/java/util/Properties.html
+ * #load(java.io.Reader). Essentially, this class assigns a [filePath] to a [Properties] instance
+ * and provides convenience, yet restricted, access to the latter.
+ *
```

Comment:
Please also mention that is a "translation" of the PropertyFiles Kotlin class from the Java Base Generator and add a TODO comment that both classes should be merged in a common JAR archive in the future to prevent code duplication and decrease maintenance.